data/json/items/gun/762.json

```diff
@@ -213,7 +215,97 @@
       [ "underbarrel mount", 1 ]
     ],
     "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "akmag30", "akmag10", "akmag20", "akmag40", "akdrum75" ] } ],
-    "melee_damage": { "bash": 8 }
+    "melee_damage": { "bash": 6 }
+  },
+  {
+    "id": "draco_mini",
+    "//": "Based off of https://www.centuryarms.com/mini-draco-pistol.html",
+    "copy-from": "pistol_base",
+    "looks_like": "draco",
+    "type": "GUN",
+    "name": { "str": "Small AK pistol" },
+    "description": "Essentially a semi-auto AK with a short barrel and no stock, this unwieldy \"pistol\" has seen some popularity with civilian shooters and gangbangers alike.  This one has a 7.75\" barrel",
```

Comment:
Can you adjust this description to match the others? I prefer your new description to the old one.

Reply:
Sure